ui/src/app/components/help-sidebar.tsx

```diff
@@ -237,6 +250,19 @@ const icons = (helpContentKey: string) => {
       style: {fontSize: '20px', marginLeft: '3px'},
       tooltip: 'Annotations',
     }];
+  if (enableCustomRuntimes && canWrite(workspaceAccessLevel)) {
```

Comment:
Adding a test case which demonstrates that this is only shown for writable workspaces would be nice, to exercise this branch